### DISCUSSION TITLE
Counting the number of switches

### DISCUSSION THREAD

**Discussion by mmmuhan** [theislab/scvelo#574] [Q&A] [answered]
Hi,

How can we use scvelo to count the number of switches and plot the figure like Fig.3(e) in the paper?

I saw a similar question at https://github.com/theislab/scvelo/issues/159#issuecomment-591068809_. but didn't see the update. Apologies if I have missed an explanation.

Any help would be appreciated!

**Comment by WeilerP** [comment, marked as answer]
@mmmuhan, you can find the notebooks to generate the figures [here](https://github.com/theislab/scvelo_notebooks). The one you are looking for should be [this](https://github.com/theislab/scvelo_notebooks/blob/master/vignettes/Fig3_pancreas.ipynb) one.
If you are also interested in the probabilistic method @VolkerBergen mentioned in #159, you can check out [CellRank](https://cellrank.readthedocs.io/en/stable/).

**Reply by mmmuhan** [reply]
Got it, thank you so much for your help!!
